front/src/views/Products/ProductsView.tsx

```diff
@@ -304,6 +318,9 @@ function Products() {
         accessor: "instockItemsCount",
         id: "instockItemsCount",
         disableFilters: true,
+        Cell: ({ row }: CellProps<ProductRow, any>) => (
+          <>{(row.original.instockItemsCount || 0) + (row.original.transferItemsCount || 0)}</>
```

Comment:
@pylipp I'd actually rather do this in the data transformation. will fix it in the table view PR